src/ontology/mondo-edit.obo

```diff
@@ -9654,7 +9654,7 @@ replaced_by: MONDO:0019347
 [Term]
 id: MONDO:0000732
 name: combined oxidative phosphorylation deficiency
-def: "A mitochondrial oxidative phosphorylation disorder in which multiple mitochondrial respiratory chain complexes." [https://orcid.org/0000-0002-6601-2165]
+def: "A mitochondrial oxidative phosphorylation disorder in which multiple mitochondrial respiratory chain complexes are affected." [https://orcid.org/0000-0002-5002-8648, https://orcid.org/0000-0002-6601-2165]
```

Comment:
This definition doesn't make much sense to me, especially when looking at the children and how each are associated with one specific gene. 

Are multiple complexes affected in one disease? That would mean that in the children term, one gene affects multiple complexes.

Reply:
For what I can find, could it be that this disease is a multisystem disorder with variable manifestations resulting from a defect in the mitochondrial oxidative phosphorylation (OXPHOS) system? (involving any of the complexes)

Reply:
See "Genetic Heterogeneity of Combined Oxidative Phosphorylation Deficiency" [here](https://omim.org/entry/609060#:~:text=Genetic%20Heterogeneity%20of%20Combined%20Oxidative%20Phosphorylation%20Deficiency)

Reply:
Yes, this makes sense. I will update with the OMIM definition. Thank you, @sabrinatoro!